release.sh

```diff
@@ -1,16 +1,21 @@
 #!/bin/bash
-rm -rf ~/Downloads/pk-release/
-mkdir ~/Downloads/pk-release/
```

Comment:
Is this file required?

Reply:
Yes, it is required, because release-droid does not yet support releasing project keeper. Until https://github.com/exasol/release-droid/issues/228 is implemented we need to release manually.